### DISCUSSION TITLE
TinyLFU for NVMCache

### DISCUSSION THREAD

**Discussion by pbhandar2** [facebook/CacheLib#187] [Ideas]
Hello,

Am I correct to understand that we cannot use TinyLFU for NVMCache admission? It is only available as a 
replacement policy for the DRAM cache. 

If so, why is this the case? Is TinyLFU not efficient to use in NVMCache? 

Pranav

**Comment by pbhandar2** [comment]
I think it is because TinyLFU would require per page statistics and CacheLib using 16MB region size in the 
NVMCache? Also when tier-1 item is being evicted, it is being written to a clean region, not necessarily evicting anything; so there is no comparison to be made of an item being admitted and evicted. I wanted to make sure I am not misunderstanding.

**Comment by haowu14** [comment]
I answered part of this in https://github.com/facebook/CacheLib/discussions/191. Let's move the discussion over there.

**Comment by pbhandar2** [comment]
Closing this discussion as it has been inactive. If this is still relevant, then please create a new discussion post and we will get back to you as soon as we can. Thank you!
